src/main/java/com/system/billingsystem/entities/InvoiceProduct.java

```diff
@@ -24,9 +23,9 @@ public InvoiceProduct() {}
     @Override
     public String toString(){
         return ("InvoiceProduct { " +
-                ", invoiceProduct_id: " + this.invoiceproduct_id +
-                ", invoice_id: " + ((this.invoice == null)? null:this.invoice.getInvoice_id()) +
-                ", product_id: " + ((this.product == null)? null: this.product.getProduct_id()) +
+                ", invoiceProduct_id: " + this.invoiceproductId +
+                ", invoice_id: " + ((this.invoice == null)? null:this.invoice.getInvoiceId()) +
+                ", product_id: " + ((this.product == null)? null: this.product.getProductId()) +
                 ", amount: " + this.amount +
```

Comment:
amount should be BigDecimal

Reply:
InviceProduct.amount referees a quantity of some product. 
No have sense on have 1,5 of product. 
Should I change the name of the variable?

Reply:
Yes, it is not `amount` but `count` then.